packages/file_selector/file_selector_android/example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 33
```

Comment:
This seems too high, doesn't it?

Reply:
No, it's the latest (android 13), and it's only intended for example and testing. Compileversion does not affect our target version. It's a compiler issue between the latest flutter compileSdkVersion and mocks